__tests__/app.test.js

```diff
@@ -274,4 +268,69 @@ describe("/api/articles/:article_id/comments", () => {
             });
         })
     });
+
+    describe("POST", () => {
+        test("Returns 'Status: 204' with empty object if sent empty request body", () => {
+            return request(app).post('/api/articles/1/comments')
+            .send()
+            .expect(204)
+            .then(({ body }) => {
+                expect(body).toEqual({})
+            });
+        });
+        test("Returns 'Status: 201' with the comment object that has been added", () => {
+            return request(app).post('/api/articles/1/comments')
+            .send({
+                body: "Mitch is cool",
+                username: "Sian"
+              })
+            .expect(201)
+            .then(({ body }) => {
+                expect(body.commentPosted).toHaveProperty("article_id", 1);
+                expect(body.commentPosted).toHaveProperty("author", "butter_bridge");
```

Comment:
Should the author not be Sian?

Reply:
Ah yes thank you! I got a bit confused because the comment database had key 'author' instead of 'username' so thought it must be the article author, but just looked again and 'author' references 'username' from the users table so they're the same thing *slaps head* Will change - thanks Rose :)